parseAPI/src/IA_power.C

```diff
@@ -312,7 +312,7 @@ bool IA_power::isReturnAddrSave(Address& retAddr) const
 bool IA_power::isReturn(Dyninst::ParseAPI::Function * context, Dyninst::ParseAPI::Block* currBlk) const
 {
   /* Check for leaf node or lw - mflr - blr pattern */
-  if (curInsn().getCategory() != c_ReturnInsn) {
+  if (curInsn().isReturn()) {
```

Comment:
This is missing a not.

Reply:
@hainest
It seems like this check is removed in #1929.
But I'm not sure if you moved this logic somewhere else in that PR.
If you did, please let me know so I can include the fix in #2074

Reply:
That was checking for idioms in ancient PowerPC and isn't needed anymore.